src/App.js

```diff
@@ -1,13 +1,17 @@
 import React from "react";
-
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer const ContactInfo = {["123 Fake Street, London, E1 4UD", "hello@fakehotel.com", "0123 456789"]} />
```

Comment:
This is not the right syntax for passing a prop into a react component. Could you correct it?